tests/test_crypt.py

```diff
@@ -60,15 +60,22 @@ def test_succeeds(self):
   def test_without_openssl(self):
     import imp
     imp_find_module = imp.find_module
+    orig_sys_path = sys.path
     def find_module(module_name):
       raise ImportError('No module named %s' % module_name)
     try:
+      for m in list(sys.modules):
```

Comment:
`sys.modules` doesn't like being replaced (or, at least, it would require a bit more fanciness). i'm leaving as-is, since it's "just a test". `;)`